runtime/gc_modron_startup/mminit.cpp

```diff
@@ -2918,6 +2918,13 @@ gcInitializeDefaults(J9JavaVM* vm)
 		goto error;
 	}
 
+#if defined(J9VM_GC_VLHGC) || defined(J9VM_GC_GENERATIONAL)
+	/* if thread library isnt assigning numa affinity, we won't either */
+	if ((omrthread_lib_get_flags() && J9THREAD_LIB_FLAG_DISABLE_DEFAULT_AFFINITY) !=0 ) {
```

Comment:
```suggestion
	if ((omrthread_lib_get_flags() && J9THREAD_LIB_FLAG_DISABLE_DEFAULT_AFFINITY) != 0) {
```

Reply:
Why it `&&` not `&` ? IMO it should be `if (J9THREAD_LIB_FLAG_DISABLE_DEFAULT_AFFINITY == (omrthread_lib_get_flags() && J9THREAD_LIB_FLAG_DISABLE_DEFAULT_AFFINITY))`